app/views/expenses/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_with(model: expense, url: group_expenses_path(@group)) do |form| %>
+  <% if expense.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(expense.errors.count, "error") %> prohibited this expense from being saved:</h2>
+
+      <ul>
+        <% expense.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= form.text_field :name, required: true, placeholder: "Expense name"  %>
+  </div>
+
+  <div class="field">
+    <%= form.number_field :amount, required: true, placeholder: "Amount" %>
+  </div>
+
+
+  <div class="actions">
+    <%= form.submit "Save" %>
+  </div>
```

Comment:
### Add a new transaction page

- [ ]  According to the project requirements, When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category and not the Home Page. Kindly correct this as well. ✍🏻 
- [ ] It's one of the project requirements that The user fills out a form to create a new transaction or expense with these mandatory parameters:
* name (mandatory)
* amount (mandatory)
* categories (mandatory at least one). 
- [ ] Kindly include the categories input select field to the expenses form or something similar 👍🏻

Reply:
```rb
<section class="add-expense">

<div class="nav">
<h5>Add Expense</h5>
</div>
<%= render "form", expense: @expense %>
<br>
<%= link_to "", @group, class: 'back'%>
<%= link_to "", group_path(@group), class: 'back' %>
</section>
```

Reply:
```erb
<section class="add-expense">
  <div class="nav">
    <h5>Add Expense</h5>
  </div>

  <%= form_with(model: [@group, @expense], url: group_expenses_path(@group)) do |form| %>
    <% if @expense.errors.any? %>
      <div style="color: red">
        <h2><%= pluralize(@expense.errors.count, "error") %> prohibited this expense from being saved:</h2>

        <ul>
          <% @expense.errors.each do |error| %>
            <<li><%= error.full_message %></li>
          <% end %>
        </ul>
      </div>
    <% end %>

    <div class="field">
      <%= form.text_field :name, required: true, placeholder: "Expense name"  %>
    </div>

    <div class="field">
      <%= form.number_field :amount, required: true, placeholder: "Amount" %>
    </div>

    <div class="field">
      <%= form.label :categories %>
      <%= form.collection_select(:group_ids, current_user.groups, :id, :name, { selected: @group.id }, { multiple: true, class: "form-control" }) %>
    </div>

    <div class="actions">
      <%= form.submit "Save" %>
    </div>
  <% end %>

  <br>

  <%= link_to "", group_path(@group), class: 'back' %>
</section>
```